test/helpers/artifacts.js

```diff
@@ -1,50 +1,51 @@
-export const RootChain = artifacts.require('RootChain')
-export const RootChainProxy = artifacts.require('RootChainProxy')
-export const Registry = artifacts.require('Registry')
-export const Governance = artifacts.require('Governance')
-export const GovernanceProxy = artifacts.require('GovernanceProxy')
-export const StakeManager = artifacts.require('StakeManager')
-export const StakeManagerProxy = artifacts.require('StakeManagerProxy')
-
-export const StakingInfo = artifacts.require('StakingInfo')
-export const StakingNFT = artifacts.require('StakingNFT')
-export const ValidatorShare = artifacts.require('ValidatorShare')
-export const StakeManagerTest = artifacts.require('StakeManagerTest')
-export const ValidatorShareFactory = artifacts.require('ValidatorShareFactory')
-
-export const DepositManager = artifacts.require('DepositManager')
-export const DepositManagerProxy = artifacts.require('DepositManagerProxy')
-export const Drainable = artifacts.require('Drainable')
-export const WithdrawManager = artifacts.require('WithdrawManager')
-export const WithdrawManagerProxy = artifacts.require('WithdrawManagerProxy')
-export const StateSender = artifacts.require('StateSender')
-
-export const ERC20Predicate = artifacts.require('ERC20Predicate')
-export const ERC721Predicate = artifacts.require('ERC721Predicate')
-export const MintableERC721Predicate = artifacts.require('MintableERC721Predicate')
-export const MarketplacePredicate = artifacts.require('MarketplacePredicate')
-export const MarketplacePredicateTest = artifacts.require(
-  'MarketplacePredicateTest'
-)
-export const TransferWithSigPredicate = artifacts.require(
-  'TransferWithSigPredicate'
-)
-
-// tokens
-export const MaticWETH = artifacts.require('MaticWETH')
-export const TestToken = artifacts.require('TestToken')
-export const DummyERC20 = artifacts.require('DummyERC20')
-export const RootERC721 = artifacts.require('RootERC721')
-export const ERC721PlasmaMintable = artifacts.require('ERC721PlasmaMintable')
-export const ExitNFT = artifacts.require('ExitNFT')
-
-// child chain
-export const childContracts = {
-  Marketplace: artifacts.require('Marketplace'),
-  ChildChain: artifacts.require('ChildChain'),
-  ChildERC20: artifacts.require('ChildERC20'),
-  ChildERC721: artifacts.require('ChildERC721'),
-  ChildERC721Mintable: artifacts.require('ChildERC721Mintable'),
-  MRC20: artifacts.require('MRC20'),
-  TestMRC20: artifacts.require('TestMRC20')
-}
+export const RootChain = artifacts.require('RootChain')

```

Comment:
All these lines are both red and green, is this a whitespace change? Also in `DepositManager.test.js`

Reply:
![image](https://user-images.githubusercontent.com/38380419/79862113-c5ffdb80-83ff-11ea-9094-d5f84425422d.png)
It seems like, i'm not sure what "<<" represents here (it is new addition in my file).